apps/balancer-tools/src/components/Table.tsx

```diff
@@ -118,14 +97,19 @@ function BodyCell({
   children,
   customWidth,
   padding = "p-4",
-}: React.PropsWithChildren<{ customWidth?: string; padding?: string }>) {
-  useTableContext();
+  colSpan = 1,
```

Comment:
I think our pattern has been to just pass classNames and let who uses it choose how to style it. We can then merge preferring the passed params using tw-merge (think I saw we were using it and that it's working well)